examples/custom-errors/custom-errors.go

```diff
@@ -24,15 +24,13 @@ func (e *argError) Error() string {
 
 func f(arg int) (int, error) {
 	if arg == 42 {
-
```

Comment:
Can you revert these changes? It's probably done by your editor (`tools/build` doesn't modify the `.go` input files).

These empty lines help with LHS comment layout

Reply:
Sorry about that. Added back these empty lines. PTAL